### PR TITLE
[Term Entry] PHP `array_map()`

### PR DESCRIPTION
**Pull request by manviii27**
### Description

 Added a new entry in PHP Arrays about `array_map` function. 
File Location : ``docs/content/php/concepts/arrays/terms/array-map/array-map.md``

### Type of Change

- Adding a new entry

### Checklist

- [ ✓] All writings are my own.
- [✓ ] My entry follows the Codecademy Docs style guide.
- [ ✓] My changes generate no new warnings.
- [ ✓] I have performed a self-review of my own writing and code.
- [ ✓] I have checked my entry and corrected any misspellings.
- [ ✓] I have made corresponding changes to the documentation if needed.
- [ ✓] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ ✓] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ ✓] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/php/concepts/arrays/terms/array-map/array-map.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  for ($i = 0; $i < count($array1); $i++) {
+    $result[] = $array1[$i] + $array2[$i];
+  }
+  return $result;
+}
+
+$array1 = array(1, 2, 3, 4, 5);
+$array2 = array(6, 7, 8, 9, 10);
+$added_arrays = array_map(add_arrays, $array1, $array2);
+print_r($added_arrays);
```
Comment (caupolicandiaz) [inline content/php/concepts/arrays/terms/array-map/array-map.md:71]:
```suggestion
function addElements($x, $y) {
  return  $x + $y;
}

$array1 = array(1, 2, 3, 4, 5);
$array2 = array(6, 7, 8, 9, 10);
$added_arrays = array_map('addElements', $array1, $array2);
print_r($added_arrays);
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@manviii27 I've completed my initial review. Please update the entry, and tag me when you've finished making changes. Thanks

**Comment by manviii27** [comment]
@caupolicandiaz I've done making the changes

File: content/php/concepts/arrays/terms/array-map/array-map.md (hunk 1 above)
Comment (manviii27) [reply content/php/concepts/arrays/terms/array-map/array-map.md]: @caupolicandiaz I've made all the changes, please check. Thank you.

**Review by caupolicandiaz — APPROVED** [review]
@manviii27 Looks good. This is ready for a second review.

File: content/php/concepts/arrays/terms/array-map/array-map.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+$new_array = array_map('double', $array);
+print_r($new_array);
+
+?>
+```
+
+The above code results in the following output:
+
+```shell
+[2, 4, 6, 8, 10]
```
Comment (SSwiniarski) [inline content/php/concepts/arrays/terms/array-map/array-map.md:50]:
```suggestion
Array
(
    [0] => 2
    [1] => 4
    [2] => 6
    [3] => 8
    [4] => 10
)

```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@manviii27 I did a first review and left a comment.

**Comment by manviii27** [comment]
@SSwiniarski I've made the suggested changes, please check. Thank you

**Review by SSwiniarski — APPROVED** [review]
@manviii27 Looks good.

**Pull request merged**